documentation/ingestion_server/guides/upgrade.md

```diff
@@ -0,0 +1,114 @@
+# Manual index upgrade runbook
+
+A manual index upgrade is similar to a data-refresh except for two key
+differences.
+
+- Each step of the process, from index creation, filtered index creation and
+  then promotion is done manually by SSH-ing into the ingestion server and
+  indexer workers.
+- It is faster than a complete data-refresh as there is no transfer of data from
+  the catalog database to the API database.
+
+## Steps
+
+### Staging deployment
+
+1. [Deploy the ingestion server](/ingestion_server/guides/deploy.md) to staging.
+   This step ensures that the latest schema will be used for the new indices.
+
+2. Determine the real names of the indexes behind the following aliases.
+
+   - `image`
+   - `image-filtered`
+   - `audio`
+   - `audio-filtered`
+
+   This information is useful to know what index to use if a rollback is needed
+   and to know what index to delete once the upgrade is complete. You can use
+   [Elasticvue](https://elasticvue.com) for this.
+
+   ```{tip}
+   In staging, the filtered indices most likely also point to the default index.
+   ```
+
+3. Perform [reindexing](/ingestion_server/reference/task_api.md#reindex) of all
+   media types. Let's say you use the suffix `abcd` for these indices. New
+   indices for each media type, like `image-abcd` and `audio-abcd`, will be
+   created.
+
+   ```{caution}
+   Staging indices are supposed to be smaller and should not the full size of
```

Comment:
Seems like a word is missing here between `not` and `the?

Reply:
Implemented in 66134fa.